database/models/sensors.js

```diff
@@ -5,38 +5,67 @@ sql.setDialect('postgres');
 var databaseP = require('../management/databaseClientP');
 var getRandomName = require('pokemon-names').random;
 
-var sensors = require('../management/declarations.js').sensors;
-var sensorCache = 'RIEN';
+var sensorTable = require('../management/declarations.js').sensors;
+var outputTable = require('../management/declarations.js').outputs;
 
 module.exports = {
-    cache: function(){
-        return sensorCache;
-    },
     create: function (data) {
-        sensorCache = 'TEST';
 
+        var sensorData = {};
+        var types = [];
+
+        Object.keys(data).forEach(function(key){
```

Comment:
Hmm... This is a bit brittle. What if both tables have keys that overlap?

Maybe do:
````js
var types = data.outputs;
// then iterate over sensor-specific keys via the sensorTable from declatations.js
var sensorData = Object.keys(sensorTable).reduce(function(acc, k){
    acc[k] = data[k];
    return acc;
}, {})
````

Not sure Object.keys(sensorTable) is the proper way to iterate. console.log it beforehand and see what's in there. I know there is a way to get the exact list.